deepchecks/checks/performance/regression_bias.py

```diff
@@ -0,0 +1,68 @@
+"""The regression_error_bias check module."""
+import matplotlib.pyplot as plt
+from sklearn.base import BaseEstimator
+from sklearn.metrics import mean_squared_error
+
+from deepchecks import CheckResult, Dataset, SingleDatasetBaseCheck, ConditionResult
+from deepchecks.utils.metrics import ModelType, task_type_validation
+from deepchecks.utils.strings import format_number
+
+__all__ = ['RegressionBias']
+
+
+class RegressionBias(SingleDatasetBaseCheck):
```

Comment:
Let's call it systematic error rather than bias (because bias raises the connotation of fairness)
```suggestion
class RegressionSystematicError(SingleDatasetBaseCheck):
```